### PR TITLE
Cherry-pick recent commits from GSL regional_workflow repo

### PR DESCRIPTION
**Pull request by chan-hoo**
## DESCRIPTION OF CHANGES:

- Cherry-pick the recent three commits from the `feature/RRFS_dev1` branch of the NOAA-GSL regional workflow repository.
- Update the PR template.

#### On machines/platforms: 

- [x] WCOSS2
- [ ] Hera
- [x] Orion
- [ ] Jet

#### Test cases:

- [x] Non-DA engineering test
- [ ] Sample script:

## ISSUE:

- Fixes the issue(s) mentioned in #17

### CODE REVIEW

**Comment by BenjaminBlake-NOAA** [comment]
@chan-hoo  These changes look good to me.  Since you tested on WCOSS2 and Orion, I can test the changes on Jet and then submit my approval.

**Review by MatthewPyle-NOAA — COMMENTED** [review]
Just one question about the exregional_run_post.sh script  chnge

File: scripts/exregional_run_post.sh
[the remark this answers is not in the document]
Comment (chan-hoo) [reply scripts/exregional_run_post.sh:1]: @MatthewPyle-NOAA, the original one failed in my second test on WCOSS2. Once I change it to the one in this PR, it ran successfully. I copied this relatively short APRUN command line from the UFS SRW App (https://github.com/ufs-community/ufs-srweather-app/blob/f9696e1bcc4295556e679e67e4ab2fb133dde2a8/ush/machine/wcoss2.yaml#L16).

[the remark this answers is not in the document]
Comment (BenjaminBlake-NOAA) [reply scripts/exregional_run_post.sh:1]: @chan-hoo  What was the error?  Since the real-time parallel is using the longer APRUN line we should try and figure out how to resolve it.

**Comment by BenjaminBlake-NOAA** [comment]
I successfully tested these changes on Jet.  Once we figure out why the longer APRUN command doesn't work on WCOSS2 I will approve this PR.

File: scripts/exregional_run_post.sh
[the remark this answers is not in the document]
Comment (chan-hoo) [reply scripts/exregional_run_post.sh:1]: @BenjaminBlake-NOAA, you can find a log file here: /lfs/h2/emc/lam/noscrub/chan-hoo.jeon/rrfs_build/expt_dirs/test_nonDA_community/log/20190615/00/run_post_f001_2019061500.log.0
```
Cannot place all ranks on node list
```

[the remark this answers is not in the document]
Comment (chan-hoo) [reply scripts/exregional_run_post.sh:1]:
```
    export OMP_NUM_THREADS=1
    export MP_IOAGENT_CNT=all
    export MP_IO_BUFFER_SIZE=8M
    export MP_BINDPROC=NO
    export MP_SHARED_MEMORY=yes
    export FI_OFI_RXM_SAR_LIMIT=3145728
    export OMP_STACKSIZE=1G
    ncores=$(( NNODES_RUN_POST*PPN_RUN_POST))
    APRUN="mpiexec -n ${ncores} -ppn ${PPN_RUN_POST} --cpu-bind core --depth ${OMP_NUM_THREADS}"
```

```
NNODES_RUN_POST="2"
PPN_RUN_POST="24"
```

[the remark this answers is not in the document]
Comment (chan-hoo) [reply scripts/exregional_run_post.sh:1]: xml file:
```
<!ENTITY RESOURCES_RUN_POST "<nodes>2:ppn=24:tpp=1</nodes>">
```

[the remark this answers is not in the document]
Comment (BenjaminBlake-NOAA) [reply scripts/exregional_run_post.sh:1]: These are the corresponding resources for the CONUS and NA real-time parallels, which request more tasks/cores and seem to work with the longer APRUN line:

CONUS: <!ENTITY RESOURCES_RUN_POST "<nodes>1:ppn=128:tpp=1</nodes>">
NA: <!ENTITY RESOURCES_RUN_POST "<nodes>4:ppn=32:tpp=4</nodes>">

[the remark this answers is not in the document]
Comment (BenjaminBlake-NOAA) [reply scripts/exregional_run_post.sh:1]: Not sure why some of it was cut off, let me try again:

CONUS: <nodes>1:ppn=128:tpp=1</nodes>
NA: <nodes>4:ppn=32:tpp=4</nodes>

**Comment by ShunLiu-NOAA** [comment]
Chan-hoo,

It might also be the system issue with using a dead node. I did see DOGWOOD
had some disk issues over the weekend.

Shun

File: scripts/exregional_run_post.sh
[the remark this answers is not in the document]
Comment (chan-hoo) [reply scripts/exregional_run_post.sh:1]: I think we can resolve this issue by adding "-l place=excl". It works well on my test.

[the remark this answers is not in the document]
Comment (BenjaminBlake-NOAA) [reply scripts/exregional_run_post.sh:1]: @chan-hoo  Good catch!  "-l place=excl" is a reasonable solution to me.  I have used that setting with various jobs.

**Review by BenjaminBlake-NOAA — APPROVED** [review]

**Review by MatthewPyle-NOAA — APPROVED** [review]
Looks good now that the post script mystery has been resolved.
